src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/Methodologies/MethodologyServiceTests.cs

```diff
@@ -807,13 +812,20 @@ public async Task GetMethodology()
                 AlternativeTitle = "Alternative title"
             };
 
+            var methodologyStatus = new MethodologyStatus
+            {
+                MethodologyVersion = methodologyVersion,
+                InternalReleaseNote = "Test approval",
+            };
```

Comment:
Missing `ApprovalStatus` here?

Reply:
I've added it, but I don't think this is strictly necessary as it gets the methodology version's status from the `MethodologyVersion` object, not from the `MethodologyStatus`. The `MethodologyStatus.ApprovalStatus` isn't unused by `GetMethodology`.

Reply:
> I see that now that we have MethodologyStatus table the MethodologyVersion.Status can differ

I did have the thought that given that we're removing `InternalReleaseNote` from `MethodologyVersion` because it's now stored in `MethodologyStatus`, you could make an argument to do the same for approval status. 

With that said, I'm not personally of the opinion that we should remove `MethodologyVersion.Status` - I think the additional confusion wouldn't make it worth it.